### PR TITLE
Updated npc Jorge, Yasir, Black Bert and some others

### PR DESCRIPTION
**Pull request by oualid6496**
-- Npc Yasir
- Updated with the latest 12.20 items

-- Npc Jorge
- Removed unnecessary repetitive code
- Added Pigeon Trophy, Loremaster Doll, Dragon Goblet & Jade Amulet Replica's
Credits to Rick for fixing a { } mistake that I made

-- Npc Black Bert
- Added Sceptre of Sun and Sea + Ivory Lyre

-- Others
- Added Net + Hand Auger

### CODE REVIEW

**Comment by Costallat** [comment]
- Build: :thumbsup:
- Server: :thumbsup:
- Test: :exclamation: (can't buy Ahmet.xml new itens, stopped test here)

**Comment by oualid6496** [comment]
> * Build: 👍
> * Server: 👍
> * Test: ❗️ (can't buy Ahmet.xml new itens, stopped test here)

Probably because the items are not updated yet.

**Comment by oualid6496** [comment]
Added the items to items.xml that were previous added to the npc's shouldn't bug now.

File: data/npc/Yasir.xml [hunk 1]
```diff
@@ -188,9 +193,12 @@
 			glob of tar, 9968, 30;
 			gloom wolf fur, 24663, 70;
 			glowing rune, 32217, 350;
```
Comment (liontts) [inline data/npc/Yasir.xml]: The correct id is: 33317

**Review by liontts — CHANGES_REQUESTED** [review]
The item name half-digested stones is also incorrect, if it is still possible to correct this pull request.
The correct id is: 30604

File: data/npc/Yasir.xml (hunk 1 above)
Comment (oualid6496) [reply data/npc/Yasir.xml]: fixed both ids

**Comment by majestyotbr** [comment]
Thanks for your contribution!

**Pull request merged**